plugins/cpp_metrics/parser/src/cppmetricsparser.cpp

```diff
@@ -107,11 +123,158 @@ void CppMetricsParser::functionParameters()
   });
 }
 
+void CppMetricsParser::lackOfCohesion()
+{
+  util::OdbTransaction {_ctx.db} ([&, this]
+  {
+    // Simplify some type names for readability.
+    typedef std::uint64_t HashType;
+
+    typedef odb::query<model::CohesionCppFieldView>::query_columns QField;
+    const auto& QFieldTypeHash = QField::CppMemberType::typeHash;
+
+    typedef odb::query<model::CohesionCppMethodView>::query_columns QMethod;
+    const auto& QMethodTypeHash = QMethod::CppMemberType::typeHash;
+    
+    typedef odb::query<model::CohesionCppAstNodeView>::query_columns QNode;
+    const auto& QNodeFilePath = QNode::File::path;
+    const auto& QNodeRange = QNode::CppAstNode::location.range;
+    
+    #ifdef DEBUG_COHESION_VERBOSE
+    std::size_t typecount =
+      _ctx.db->query_value<model::CppRecordCount>().count;
+    std::size_t typeindex = 0;
+    std::size_t checkedcount = 0;
```

Comment:
`std::size_t` is usually used as a size of something. If it's not a container that has an actual size, it is more natural to use `unsigned` instead of `size_t`.
Also, please follow the coding conventions even if these are log variables (`typeCount` etc.)

Reply:
Thanks for noticing the variable casing, that was just hasty coding on my part in places that were initially only supposed to be temporary debug-only features. But of course if we intend to keep them under `--loglevel=debug,` they will be refactored.

About `std::size_t`: I went with it because the already existing `model::CppRecordCount::count` field that I used to get the total number of types was already defined as `std::size_t`. From that, to me it followed logically that the type of a "range" and its "indexer" should match (the same way we use `std::size_t` for both the size and the indexing of a `std::vector`).